bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java

```diff
@@ -162,13 +164,34 @@
             new ProductUnit<VolumetricFlowRate>(Units.CUBIC_METRE.divide(Units.HOUR)));
     public static final Unit<VolumetricFlowRate> CUBICMETRE_PER_DAY = addUnit(
             new ProductUnit<VolumetricFlowRate>(Units.CUBIC_METRE.divide(Units.DAY)));
+    public static final Unit<DataAmount> BIT = addUnit(new AlternateUnit<DataAmount>(ONE, "bit"));
+    public static final Unit<DataAmount> MEGABIT = addUnit(MetricPrefix.MEGA(BIT));
+    public static final Unit<DataAmount> KILOBIT = addUnit(MetricPrefix.KILO(BIT));
+    public static final Unit<DataAmount> GIGABIT = addUnit(MetricPrefix.GIGA(BIT));
+    public static final Unit<DataAmount> TERABIT = addUnit(MetricPrefix.TERA(BIT));
+
+    public static final Unit<DataAmount> BYTE = addUnit(BIT.multiply(8));
+    public static final Unit<DataAmount> OCTET = BYTE;
+    public static final Unit<DataAmount> KIBIOCTET = addUnit(BinaryPrefix.KIBI(OCTET));
+    public static final Unit<DataAmount> MEBIOCTET = addUnit(BinaryPrefix.MEBI(OCTET));
+    public static final Unit<DataAmount> GIBIOCTET = addUnit(BinaryPrefix.GIBI(OCTET));
+
+    public static final Unit<DataTransferRate> BIT_PER_SECOND = addUnit(
+            new ProductUnit<DataTransferRate>(BIT.divide(Units.SECOND)));
+    public static final Unit<DataTransferRate> KILOBIT_PER_SECOND = addUnit(MetricPrefix.KILO(BIT_PER_SECOND));
+    public static final Unit<DataTransferRate> MEGABIT_PER_SECOND = addUnit(MetricPrefix.MEGA(BIT_PER_SECOND));
+    public static final Unit<DataTransferRate> GIGABIT_PER_SECOND = addUnit(MetricPrefix.GIGA(BIT_PER_SECOND));
+    public static final Unit<DataTransferRate> TERABIT_PER_SECOND = addUnit(MetricPrefix.TERA(BIT_PER_SECOND));
 
     /**
      * Add unit symbols for custom openHAB units.
      */
     static {
         // Ordered alphabetical by name
         SimpleUnitFormat.getInstance().label(BAR, BAR.getSymbol());
+        SimpleUnitFormat.getInstance().label(BIT, BIT.getSymbol());
+        SimpleUnitFormat.getInstance().label(BIT_PER_SECOND, BIT.getSymbol() + "/" + Units.SECOND.getSymbol());
```

Comment:
Any specific reason to use this notation instead of "bit/s" like in the other cases?